h2o-algos/src/main/java/hex/tree/dt/AbstractFeatureLimits.java

```diff
@@ -0,0 +1,13 @@
+package hex.tree.dt;
+
+/**
+ * Limits for one feature.
+ */
+public abstract class AbstractFeatureLimits {
```

Comment:
Is it necessary to have `Abstract` in the name? I prefer to have only CompressedNode, FeatureLimits, and SplittingRule.

Reply:
`Abstract` in the name of `abstract` class is a naming convention in Java.